azure-devops/run-tests.yml

```diff
@@ -6,15 +6,15 @@ parameters:
   type: string
 - name: targetArch
   type: string
-- name: ctestOptions
+- name: testTargets
   type: string
 - name: skipTesting
   type: boolean
 steps:
 - script: |
     call "%ProgramFiles%\Microsoft Visual Studio\2022\Preview\Common7\Tools\VsDevCmd.bat" ^
     -host_arch=${{ parameters.hostArch }} -arch=${{ parameters.targetArch }} -no_logo
-    ctest --verbose ${{ parameters.ctestOptions }}
+    ninja ${{ parameters.testTargets }}

```

Comment:
Should we use `ctest --build` here instead of invoking `ninja` directly to future-proof this?